libs/model/src/database.ts

```diff
@@ -5,15 +5,19 @@ import { buildDb } from './models';
 
 const log = logger(import.meta);
 
-export const sequelize = new Sequelize(config.DB.URI, {
+const dbURI =
+  config.NODE_ENV === 'test'
+    ? 'postgresql://commonwealth:edgeware@localhost/common_test'
+    : config.DB.URI;
```

Comment:
Tests were not working for me, in fact the authorization will make the calls to the real database if this is not there. I don't know if it happened recently, but without this my test will fail

Reply:
@kurtisassad do you have `DATABASE_URL` or `TEST_DB_NAME` defined in your `.env` file? If so then you are overriding the DB URI when running the test since this is how DB URL is set:

```
const NAME = target.NODE_ENV === 'test' ? TEST_DB_NAME || 'common_test' : 'commonwealth';
const DEFAULTS.DATABASE_URL = `postgresql://commonwealth:edgeware@localhost/${NAME}
URI: DATABASE_URL ?? DEFAULTS.DATABASE_URL```
```

Reply:
The tests run fine in my fork of your branch without your `dbURI` so I think the issue is local to you.

Reply:
removed

Reply:
I have TEST_DB_NAME=common_test but it still doesn't work. Anyways I can just add this code when I need it, as it is a more universal way to switch the database connection